src/domain/collaboration/collaboration/collaboration.service.ts

```
@@ -198,9 +199,23 @@ export class CollaborationService {
         LogContext.COLLABORATION
       );
     }
+    const calloutNameIds: string[] = compact([
+      ...collaboration.callouts?.map(callout => callout.nameID),
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)